### DISCUSSION TITLE
Support on dotnet 6.0/Windows 7,8.1

### DISCUSSION THREAD

**Discussion by yjagota** [ravendb/ravendb#15610] [Q&A] [answered]
RavenDB v5.4.100 has been moved to dotnet 7.0, [which drops support for Windows 7 & 8.1](https://github.com/dotnet/core/blob/main/release-notes/7.0/supported-os.md). So essentially, a person targeting Windows 7/8.1 is either stuck with v5.4.5 for dotnet 6.0 with no further bugfixes, or has to drop support for Windows 7/8.1. So is there any plan to maintain support for both dotnet 6.0 and dotnet 7.0 going forward? Although by the looks of it, it seems dotnet 6.0 support will be dropped, but just wanted to confirm.

Thanks.

**Comment by ppekrol** [comment, marked as answer]
Hi @yjagota 

As you noticed MS dropped support (incl. extended) for both those operating systems, but our RavenDB 5.2 will continue working on .NET 6 and it is fully supported till the end of July 2023 and since it is LTS, it will be eligible for Extended Support as well.
